### PR TITLE
docs(guidelines): add guidelines

### PR DESCRIPTION
**Pull request by clementgig**
- "1) Prerequisites" section : Add mention to public activity material
- "3) Team/Maintainer Must" section : Add guidelines
- Add new section "5) Repository end of life"
- Fix Markdown warnings

### CODE REVIEW

**Review by fdescamps — COMMENTED** [review]
OK

**Comment by fdescamps** [comment]
OK

**Review by fdescamps — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -59,11 +60,26 @@ Please consider to be **Open Source First**:
 
 ## 3) Team/Maintainer Must
 
-- Use and force two factor Authentication everywhere it is possible: github, npm, nuget, etc.
-- Tokens must be kept private
+Checklist for repository creation:
+
+- Use snake-case pattern to name your repository
```
Comment (guillaume-chervet) [inline README.md]: Repository name convention should more depends of the language.
SlimFaas is more powerful than slimfaas.

I'am not for a rule for that, you cannot manage fork Repository name.

Comment (clementgig) [reply README.md]: Good point, I removed the name convention rule.

**Review by guillaume-chervet — APPROVED** [review]

**Comment by guillaume-chervet** [comment]
Thank you @clementgig for your help.

Are you ready to merge it?

**Comment by clementgig** [comment]
> Thank you @clementgig for your help.
> 
> Are you ready to merge it?

Yes, i will merge now

**Pull request merged**